api-reference/10 UI Components/dxCardView/9 Types/RemoteOperations/grouping.md

```diff
@@ -5,7 +5,7 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether grouping must be performed on the server side.
```

Comment:
Can you please add note that it is used only for header filter, like in TreeList's doc? https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/remoteOperations/#grouping